src/OpenCensus/Trace/Span.cs

```diff
@@ -77,12 +77,18 @@ public sealed class Span : SpanBase
             }
         }
 
+        /// <inheritdoc/>
         public override string Name
         {
             get
             {
                 return this.name;
             }
+
+            set
+            {
+                this.name = value;
```

Comment:
maybe it's a style preference, but why adding "this." where it's not required?

Reply:
it's stylecop rule. There are two major camps: `_name` for fields and `this.name` for fields. I just use whatever default in Stylecop, which is `this.name`